Colours_And_Fonts_style_v1.css

```diff
@@ -50,7 +115,7 @@ p {
 
 .header {
   box-sizing: border-box;
-  background-color: #2c7da0;
+  background-color: #3750A5;

```

Comment:
I changed the colour from a dark geeny blue colour to a navy blue as I wanted to incoperate the Tawa College colours so end users know this website is made for Tawa College. I also thought the first wasn't as great to look at as it was too dark.